bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/VoiceManager.java

```diff
@@ -251,4 +251,12 @@ void startDialog(@Nullable KSService ks, @Nullable STTService stt, @Nullable TTS
      */
     Set<Voice> getAllVoices();
 
+    /**
+     * Returns the default voice used for TTS.
+     *
+     * @return the default voice or null, if no default voice is configured or if a default is configured, but no
+     *         according service is found
+     */
+    @Nullable
+    Voice getDefaultVoice();
```

Comment:
Just noticed that this is API breaking. One option would be to give a default implementation here (returning `null` I guess) or add a new service which serves the default voice only.

Reply:
See my [comment on the AudioManager](https://github.com/eclipse/smarthome/pull/6043#discussion_r209846808) - I do not consider this API breaking.